<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Стобцы</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Строки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_search">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_column">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите элемент:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_sort">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_random">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Рандом</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_row">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Report">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>551</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bubble</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quick</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Comb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gnome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bogosort</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>121</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Поиск:</string>
    </property>
    <widget class="QPushButton" name="Button_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_mid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mid</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_max">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_delete">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Удалить Дубликаты</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
